src/main/minio.js

```diff
@@ -1421,6 +1421,61 @@ export class Client {
     this.makeRequest({method, bucketName, objectName}, '', 204, '', false, cb)
   }
 
+  // Remove all the objects residing in the objectsList.
+  //
+  // __Arguments__
+  // * `bucketName` _string_: name of the bucket
+  // * `objectsCh` _string_: channel of objects
```

Comment:
Typo, rename `objectCh` to `objectsList`

Reply:
*oops ! will change it